build/templates/venv-commons

```diff
@@ -42,7 +42,7 @@ $KONG_VENV/openresty/site/lualib/?.lua;$KONG_VENV/openresty/site/lualib/?.ljbc;\
 $KONG_VENV/openresty/site/lualib/?/init.lua;$KONG_VENV/openresty/site/lualib/?/init.ljbc;\
 $KONG_VENV/openresty/lualib/?.lua;$KONG_VENV/openresty/lualib/?.ljbc;\
 $KONG_VENV/openresty/lualib/?/init.lua;$KONG_VENV/openresty/lualib/?/init.ljbc;\
-$KONG_VENV/openresty/luajit/share/luajit-2.1.0-beta3/?.lua"
+$KONG_VENV/openresty/luajit/share/luajit-2.1/?.lua"
```

Comment:
I was curious about this change, so I went to inspect my local build. I can see that I actually have both directories in kong master:

```
$ ls bazel-kong/bazel-out/k8-fastbuild/bin/external/openresty/luajit/share/
drwxr-xr-x. michaelm michaelm 4.0 KB Wed Oct 18 13:43:19 2023  lua
drwxr-xr-x. michaelm michaelm 4.0 KB Thu Jan 18 11:06:14 2024  luajit-2.1
drwxr-xr-x. michaelm michaelm 4.0 KB Wed Oct 18 13:43:19 2023  luajit-2.1.0-beta3
drwxr-xr-x. michaelm michaelm 4.0 KB Wed Oct 18 13:43:19 2023  man
```

Can you explain more of the context? Is this just a fix for an older path that we forgot to update?

Reply:
@flrgh The LuaJIT moved to so called `2.1.ROLLING` and `-beta3` is more or less gone. Your `-beta3` is created on `2023`. so perhaps it is a leftover.